android/src/com/mapswithme/maps/Framework.java

```diff
@@ -61,6 +61,13 @@
     public boolean buildings;
   }
 
+  public static class RouteAltitudeLimits
+  {
+    public int minRouteAltitude;
```

Comment:
I had a look at m_routingSession.m_route. It's quite possible to wrap it in shared_ptr and use it as we agreed. But it takes some time. Artem ask to merge it ASAP. So let's merge it as is and implement share_prt<Route> in separate PR.